src/components/BookCard.jsx

```diff
@@ -0,0 +1,45 @@
+import '../styles/BookCard.scss';
+
+function BookCard() {
+  return (
+    <article className="book">
+      <div className="book__left">
+        <div className="book__left-description">
+          <h3 className="book__left-h3">Action</h3>
+          <h2 className="book__left-h2">The Hunger Games</h2>
+          <p className="book__left-p">Suzanne Collins</p>
+          <ul className="book__left-ul">
+            <li className="book__left-li">Comments</li>
+            <li className="book__left-li">Remove</li>
+            <li className="book__left-li">Edit</li>
+          </ul>
+        </div>
+        <div className="book__left-circle">
+          <svg
+            className="book__left-svg"
```

Comment:
- [ ] You actually did great work here 👏 . However, i'll advice you breakdown this component into two sections.

- `<Book>`: Takes the book data as a prop and renders the title and the author
  * Has a button for removing books
  * the button wouldn't remove the books *yet*, this is just a placeholder for now

- `<BookList>`: Renders a list of books using the `<Book>` component

- [ ] Consider using your `form` component here also, so you have a composite `Bookcard` component to call anywhere in your code. This is the idea of react after all, to have reusable component 👍

Reply:
Hello @Shedrack-Sunday,

When you say "into two sections" at the beginning you mean components?